boards/posix/simple_process/doc/board.rst

```diff
@@ -0,0 +1,179 @@
+interrupt, the interrupt controller may raise the interrupt immediately
+depending on interrupt priorities, masking, and locking state.
+
+Normally the resulting executable runs fully decoupled from the real host time.
+That is, simulated time will advance as fast as it can. This is desirable when
+running in a debuger or testing in batch, but not if one wants to interact
+with external interfaces which are based on the real host time.
+
+Peripherals
+***********
+
+The following peripherals are currently provided with this board:
+
+**Console/printk driver**:
+
+A driver is provided that redirects any printk write to the native
+host application's stdout.
+
+**Simple timer**:
+
+A simple timer provides the kernel with a 10ms tick.
+This peripheral driver also provides the needed functionality for this
+architecture-specific k_busy_wait().
+
+This timer, may also be configured with SIMPLE_PROCESS_SLOWDOWN_TO_REAL_TIME
+to slow down the execution to host time.
+This will provide the ilusion that the simulated time is running at the same
+speed as the host time.
+In reality, the timer will monitor how much real time
+has actually passed since boot, and when needed will pause the execution
+before raising each timer interrupt.
+Normally the Zephyr application and HW models run in very little time
+on the host CPU, so this is a good enough approach.
+
+
+**Interrupt controller**
+
+A simple yet generic interrupt controller is provided. It can nest interrupts
+and provides interrupt priorities. Interrupts can be individually masked or
+unmasked. SW interrupts are also supported.
+
+
+Important limitations
+=====================
+
+The assumption that simulated time can only pass while the CPU is sleeping
+means that there can not be busy wait loops in the application code that
+wait for something to happen without letting the CPU sleep.
+If busy wait loops do exist, they will behave as infinite loops and
+will stall the execution.
+
+As simulated time does not pass while the CPU is running, it also means no HW
+interrupts will interrupt the threads' execution unless the SW enables or
+unmasks them.
+This is intentional, as it provides a deterministic environment to develop and
+debug.
+But note that this may hide issues in the SW that may only be triggered in the
+real platform.
+
+This native port of Zephyr is not meant to, and could not possibly
+help debuggin races between HW and SW, or similar timing related issues.
+
+
+How to use it
+*************
+
+Compiling
+=========
+
+Specify the simple_process board target to build a native POSIX application:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: simple_process
+   :goals: build
+   :compact:
+
+
+Running
+=======
+
+The result of the compilation is an executable (zephyr.exe) placed in the
+zephyr/ subdirectory of the build folder.
+Run the zephyr.exe executable as you would any other Linux console application.
```

Comment:
does the developer need to do a chmod +x on zephyr.exe?

Reply:
No, it is executable already